src/analog/utils/CommandBinaryBuffer.ts

```diff
@@ -15,6 +15,7 @@ import roundNumber from '../../utils/roundNumber.js';
 import {extractBits, fillBits} from '../../utils/bitSet.js';
 import * as hardwareTypes from '../constants/hardwareTypes.js';
 import * as deviceParameters from '../constants/deviceParameters.js';
+import invertObject from '../../utils/invertObject.js';
```

Comment:
can't see this file in mr

anyway it's old implementation will not work in otto because of Object.entries